Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1076,9 +1077,11 @@ def png(self, file, width=None, height=None,
         if width is not None and height is not None:
             if self.renWin.GetSize() != (width, height):
                 user_dims = (self.canvas.bgX, self.canvas.bgY, sz[0], sz[1])
-                self.renWin.SetSize(width, height)
+                # We need to set canvas.bgX and canvas.bgY before we do renWin.SetSize
+                # otherwise, canvas.bgX,canvas.bgY will win
```

Comment:
> if SetSize triggers something which set bgX and bgY in some cases

No, this SetSize triggers code that, in certain cases, sets the size of the size of the window to bgX, bgY rather than the parameters passed to SetSize. The test that fails with the order reversed is vcs_test_png_set_size.